_projects/improve_diff_handling.md

```diff
@@ -41,11 +37,108 @@ corrections or gramatical corrections.
 In addition Bears will be able to offer templated patches, e.g. to offer
 variable renamings while the user should be asked for the new variable name.
 
+#### Sample mockups
+
+Source: https://docs.google.com/document/d/1BLZO2tD93yKdO7XhMb5NgJvytu-pWDtPMRidGmjQ8EY/edit?usp=sharing (You are free to use ideas discussed in the proposal).
+
+Candidates can take help of following sample mockups to understand the use cases better. The mockups discussed below however need not be strictly followed. Candidate is enocouraged than welcome to incorporate his ideas and designs.
```

Comment:
enocouraged - spelling... but, use stronger language, and gender neutral when possible:

> Candidate is expected to incorporate their own ideas and designs.